### DISCUSSION TITLE
Sakura Blossom live wallpaper not working

### DISCUSSION THREAD

**Discussion by tashi21** [rocksdanister/lively#601] [Q&A] [answered]
![Screenshot (39)](https://user-images.githubusercontent.com/79501323/125151978-a28d9180-e167-11eb-87d5-77d568b1ee58.png)

[Sakura Blossom](https://www.deviantart.com/rocksdanister/art/Sakura-Blossom-842466561)

Whenever I try using the Sakura Blossom wallpaper, the background image selected does not get displayed (as shown above). The petals continue falling. I have tried deleting and re-downloading the wallpaper. This is the only wallpaper I face this issue with. Please guide me on how to fix this. And I apologize if this message is in the wrong thread, I am quite new to this.

**Comment by rocksdanister** [comment, marked as answer]
Are you using Webview (Edge) instead of CefSharp browser in settings?

**Reply by tashi21** [reply]
Yeah, I'm using MS Edge

**Reply by rocksdanister** [reply]
Yeah this is because of CORS issue.. not much I can do till Edge implements a Schemehandler for local files.
Stick to Cefsharp.

**Reply by tashi21** [reply]
Alright, thank you!
